HW1/P2/P2b.py

```diff
@@ -1,3 +1,23 @@
+import numpy as np
 from P2 import *
 
-# Your code here
+import findspark
+findspark.init()
+
+import pyspark
+sc = pyspark.SparkContext()
+
+sc.setLogLevel('WARN')
+
+x = np.random.permutation(2000)
```

Comment:
Rather than randomly permuting and then passing in to sc.parallelize, it seems easier to use sc.partitionby and pass in a lambda function of randint. Not sure which is faster since np is quite fast.